### DISCUSSION TITLE
Floorp Browser Portable, These two problems ...

### DISCUSSION THREAD

**Discussion by ioc2e3** [Floorp-Projects/Floorp#624] [Q&A]
Traditional Chinese interface is not completely traditional Chinese ...

Mixed English, Simplified Chinese ...

Even from this ( https://addons.mozilla.org/zh-TW/firefox/addon/traditional-chinese-zh-tw-l/ ) downloading traditional Chinese language files, the same ...

...............................

in addition ...

Why doesn't Portable Version automatic update work?

**Comment by creeper-0910** [comment]
What platforms are you using the portable version on?

**Reply by ioc2e3** [reply]
Windows 10
Is 64-bit

.

**Reply by ioc2e3** [reply]
Wait a moment ...

Floorp Browser Portable cannot be updated automatically in Windows?


Is this the reason ...?

**Reply by creeper-0910** [reply]
No, Floorp Portable supports updates on Windows
I'll do some research...

**Reply by creeper-0910** [reply]
It was a configuration error, our apologies.
This problem will be fixed in the next version.
To fix it, you need to set floorp.portable.isUpdate to true in about:config.

**Reply by ioc2e3** [reply]
Portable still cannot update automatically ... 

... ( 11.7.0 version ) and ( 11.7.1 version ) have this situation

... ( in about:config to set floorp.portable.isupdate to true ) still the same

.

**Reply by creeper-0910** [reply]
Sorry, we cannot reproduce that issue...
Please download the latest version and wait for the next update.

**Reply by ioc2e3** [reply]
> 抱歉，我們無法重現問題... 請下載最新版本並等待下次更新。

( icon of circled ) No matter how press the mouse , there is no response ...
![zz1](https://github.com/Floorp-Projects/Floorp/assets/152480314/9bf41a56-45e1-4e0c-898b-c22a3ba5c621)

Even if Floorp Browser Portable updates automatically , the problem remains the same ...
![zz2](https://github.com/Floorp-Projects/Floorp/assets/152480314/8bc2eb93-4c1d-4072-9e60-ec13b0ef13ed)

.

**Reply by creeper-0910** [reply]
The button is not available in the portable version.
You will be notified when there is an update, and the next time you start the program, it will be updated automatically.

**Comment by CutterKnife** [comment]
Localization of Floorp is left to the people of Crowdin.
